shared/lib/deep-links/routes/index.ts

```diff
@@ -42,3 +43,4 @@ addRoute(perps);
 addRoute(predict);
 addRoute(rewards);
 addRoute(shield);
+addRoute(settings);
```

Comment:
We always need e2e tests for these routes (I'm sure some teams are slipping by without adding them, but they aren't supposed to!).

Reply:
Added one on top of the existing setup.